### ISSUE TITLE
npm install errors

### ISSUE THREAD

**Issue by VishalAkhouri** [arcatdmz/nextjs-with-jest-typescript#3]
Apparently `npm install` is throwing a bunch of issues that will require a version upgrades to the dependencies.

```
akhouri@Akhouris-MBP nextjs-with-jest-typescript % npm install
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.2
npm WARN node_modules/react
npm WARN   react@"^17.0.2" from the root project
npm WARN   3 more (@wojtekmaj/enzyme-adapter-react-17, react-dom, next)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"0.13.x || 0.14.x || ^15.0.0-0 || ^16.0.0-0" from enzyme-adapter-utils@1.14.0
npm WARN node_modules/enzyme-adapter-utils
npm WARN   enzyme-adapter-utils@"^1.14.0" from @wojtekmaj/enzyme-adapter-react-17@0.5.0
npm WARN   node_modules/@wojtekmaj/enzyme-adapter-react-17
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR!   peer react@"^17.0.0-0" from @wojtekmaj/enzyme-adapter-react-17@0.5.0
npm ERR!   node_modules/@wojtekmaj/enzyme-adapter-react-17
npm ERR!     dev @wojtekmaj/enzyme-adapter-react-17@"^0.5.0" from the root project
npm ERR!   2 more (react-dom, next)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^0.14 || ^15.0.0 || ^16.0.0-alpha" from airbnb-prop-types@2.16.0
npm ERR! node_modules/enzyme-adapter-utils/node_modules/airbnb-prop-types
npm ERR!   airbnb-prop-types@"^2.16.0" from enzyme-adapter-utils@1.14.0
npm ERR!   node_modules/enzyme-adapter-utils
npm ERR!     enzyme-adapter-utils@"^1.14.0" from @wojtekmaj/enzyme-adapter-react-17@0.5.0
npm ERR!     node_modules/@wojtekmaj/enzyme-adapter-react-17
npm ERR!       dev @wojtekmaj/enzyme-adapter-react-17@"^0.5.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/akhouri/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/akhouri/.npm/_logs/2021-08-20T23_09_15_509Z-debug.log
```

**Comment by VishalAkhouri** [comment]
I was able to resolve this by `npx npm-check-updates -u`
```
 @types/enzyme                        ^3.10.8  →   ^3.10.9     
 @types/jest                         ^26.0.21  →   ^27.0.1     
 @types/react                         ^17.0.3  →  ^17.0.19     
 @types/react-dom                     ^17.0.3  →   ^17.0.9     
 @wojtekmaj/enzyme-adapter-react-17    ^0.5.0  →    ^0.6.3     
 jest                                 ^26.6.3  →   ^27.0.6     
 ts-jest                              ^26.5.4  →   ^27.0.5     
 typescript                            ^4.2.3  →    ^4.3.5     
 next                                 ^10.0.9  →   ^11.1.0     

Run yarn install to install new versions.
```

and then `npm install`

**Comment by arcatdmz** [comment]
This should be resolved by the PR #4 contributed by @VishalAkhouri.
